spec/dummy/app/models/user.rb

```diff
@@ -4,5 +4,7 @@
 class User < ApplicationRecord
   has_many :posts
 
-  attr_reader :password, :file, :role
+  attr_reader :password, :file
+  # :reek:Attribute { enabled: false }
+  attr_accessor :role
```

Comment:
why did you change  `attr_reader` to `attr_reader` here?

Reply:
We need it for the last test in `spec/components/tailwinds/forms/builder_spec.rb`. Line 95